cpu/kinetis/periph/timer.c

```diff
@@ -184,6 +201,7 @@ static inline int pit_set(uint8_t dev, uint32_t timeout)
     const uint8_t ch = pit_config[dev].count_ch;
     /* Disable IRQs to minimize the number of lost ticks */
     unsigned int mask = irq_disable();
+
```

Comment:
unrelated style changes

Reply:
I've run the code through uncrustify at some point, which annoyingly doesn't always leave already committed code unchanged. I'll proof-read again.